### DISCUSSION TITLE
XTTS2 - bad output audio quality.. Parameter that allows for 44.1k audio output quality in XTTS2 ?

### DISCUSSION THREAD

**Discussion by identificator** [coqui-ai/TTS#3734] [General Q&A] [answered]
I am using `xtts_v2 `model, and when `tts.tts_to_file` is executed I am not satisfied with the result - 24kHz wav. 
The output wav has a lower bitrate and sounds poor quality. 
How can I customize tts so that it produces sound wave of original quality in 44100 sample rate. 
Is there any parameter?
Or is it in the `xtts_2 `model itself, and I need to train my own model for the original bitrate quality?
Or even if I train my own model - `coqui tts` will still produce low quality sound?

For example, there is no such problem in RVC2.

My code

```
from TTS.api import TTS
tts = TTS("tts_models/multilingual/multi-dataset/xtts_v2", gpu=True)

# generate speech by cloning a voice using default settings
tts.tts_to_file(text="It took me quite a long time to develop a voice, and now that I have it I'm not going to be silent.",
                file_path="output.wav",
                speaker_wav=["speaker.wav"],
                language="en",
                split_sentences=True
                )
```

**Comment by ghost** [comment, marked as answer]
https://discord.com/channels/1037326658807533628/1038916382970089612/1228792722207277147
lexkoro — 14.04.2024 0:43
Because it was trained on 24kHz

**Reply by identificator** [reply]
Will a model trained in 44.1 or 48 kHz be available?

**Reply by ghost** [reply]
https://discord.com/channels/1037326658807533628/1038916382970089612/1235253593993510983
E U N O I A — 01.05.2024 20:36
Coqui AI as a company shutdown, so support and updates are no longer happening until they changed their mind.

**Comment by ghost** [comment]
Try to train the model yourself with a higher sampling rate.

**Reply by identificator** [reply]
Can you advise some tutorial to make training your model on WAV datasets effectively? I've seen tutorials where you have to write a text description for each dataset-wav file, thus training the model. For example in Applio, where training a model is much easier and the result is predictable.

**Reply by ghost** [reply]
Alas, I don't understand anything about it.
